src/Controller/Admin/AdministrationBaseController.php

```diff
@@ -133,10 +133,19 @@ public function index(): ?Response
         $this->set('propertiesSecrets', $this->propertiesSecrets);
         $this->set('metaColumns', $this->meta);
         $this->set('filter', []);
-        $this->set('schema', (array)$this->Schema->getSchema($this->resourceType));
+        $this->set('schema', []);
```

Comment:
are we sure it's never used?

Reply:
admin endpoints lack schema (they are not objects). am I right?

Reply:
also resources have a schema => look in `config/schema_properties.php` - schema is fixed on resources and dynamic on objects